### DISCUSSION TITLE
[RFC] 014 - TTS & STT 二期

### DISCUSSION THREAD

**Discussion by canisminor1990** [lobehub/lobehub#480] [RFCs]
- [x] 支持本地持久化存储 https://github.com/lobehub/lobe-chat/pull/587
- [ ] 提供纯语音对话 Stream 模式 ( 参考 官方 GPT Pro )
- [ ] 考虑增加 OpenAI TTS & STT 单独 Key 配置
- [ ] 自动播放 #986 
> ref: https://github.com/lobehub/lobe-chat/discussions/483
> 可以和多 Key Feat 合并考虑 @arvinxx

**Comment by Dismosoul903** [comment]


**Reply by Dismosoul903** [reply]
@Dismosoul903 @canisminor1990

**Comment by Zeke-Wu** [comment]
marked

**Comment by Merlinvt** [comment]
Gemini also has audio input and speech + text output now. Would you consider adding those features? Because of Gemini 2.5 Pro, Gemini has become my default model provider.

https://ai.google.dev/api/live

If you do implement the voice feature for OpenAI and Gemini, please add a 'hold to speak' button too. One of the most frustrating things with Gemini Mobile on Android is that you can't hold the audio button down. One of the things I recently loved about using Google AI Studio was that you're able to record multiple voice files and then send them all at once. (Sometimes I record a message, then I get interrupted by my son. I can just click stop and start recording another message when I pick up again.) Then I can send them. It's really convenient, and I love it!
